packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -861,7 +863,8 @@ class PopupMenuButton<T> extends StatefulWidget {
 
   /// If provided, the icon used for this button.
   final Icon icon;
-
+  final double offsetX;
+  final double offsetY;
```

Comment:
Generally we'd probably just take a single `Offset` argument.

Reply:
You can use the Offset Widget instead.the code may likes this:
`new PopupMenuButton(
tooltip: "sort",
Offset(0.0,0.0)
...
)`

`final Offset offset = Offset.zero`